collect_app/src/main/java/org/odk/collect/android/application/MapboxClassInstanceCreator.kt

```diff
@@ -14,7 +14,7 @@ object MapboxClassInstanceCreator {
             "armeabi-v7a"
         )
 
-        return createMapboxMapFragment() != null && Build.SUPPORTED_ABIS.any(mapboxAbis::contains)
+        return createMapboxMapFragment() != null && mapboxAbis.contains(Build.SUPPORTED_ABIS.first())
```

Comment:
This feels very unlikely, but can we protect from `SUPPORTED_ABIS` being empty here? I guess adding `!Build.SUPPORTED_ABIS.isEmpty()` as part of the `&&` expression would do.